.docker/wp-config.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * The base configuration for WordPress
```

Comment:
`SHOULD` - I think all the comments can be removed, they're intended for new users and just clutter the file. Anything non-obvious should be documented, though.

Reply:
Removed redundant comments in https://github.com/WordPress/wordcamp.org/pull/113/commits/1e3bc2e5e0ebcc4643db21c0d98a69b94a51b6c8